### PR TITLE
Update app.js

### PR DESCRIPTION
**Pull request by josh-p-git**
Added code for Render deployment

### CODE REVIEW

**Comment by abodian** [comment]
It might be best to hold off on this pull request as I have significantly changed the backend so that it can communicate correctly with the frontend. I don't think I had set it up correctly initially. My apologies. Once my pull request (16) is merged then we can see how it conflicts?

**Review by terrybro66 — APPROVED** [review]

**Pull request closed without merging**
